smithy-jsonschema/src/test/java/software/amazon/smithy/jsonschema/JsonSchemaConverterTest.java

```diff
@@ -780,4 +781,106 @@ public void intEnumsCanBeDisabled() {
                 IoUtils.toUtf8String(getClass().getResourceAsStream("int-enums-disabled.jsonschema.v07.json")));
         Node.assertEquals(document.toNode(), expected);
     }
+
+    @Test
+    public void supportsDeprecatedTraitOnAStruct() {
+        StringShape string = StringShape.builder().id("smithy.api#String").build();
+        StructureShape shape = StructureShape.builder()
+        .id(ShapeId.from("a.b#C"))
+        .addMember(
+                MemberShape.builder().id(ShapeId.from("a.b#C$member"))
+                .target(string.getId())
+                .build()
+        )
+        .addTrait(DeprecatedTrait.builder().message(
+                        "I'm deprecated"
+                ).since("sinceVersion").build())
+        .build();
```

Comment:
```suggestion
        StructureShape shape = StructureShape.builder()
                .id(ShapeId.from("a.b#C"))
                .addMember(MemberShape.builder()
                        .id(ShapeId.from("a.b#C$member"))
                        .target(string.getId())
                        .build())
                .addTrait(DeprecatedTrait.builder()
                        .message("I'm deprecated")
                        .since("sinceVersion")
                        .build())
                .build();
```